.github/release_template.md

```diff
@@ -0,0 +1,50 @@
+<!--
+This is the template to release a new version on the App Store
+-->
+
+Release a new version
+## TASKS:
+
+Git & Code
+
+- [ ] [GIT] Create branch `release/[major].[minor].[patch]` (freeze the code)
+- [ ] [DEV] Update version number `[major].[minor].[patch]`
+- [ ] [DIS] Update [changelog](https://github.com/owncloud/ios-app/blob/master/CHANGELOG.md)
+- [ ] [QA] Design Test plan
+- [ ] [QA] Regression Test plan
+- [ ] [DOC] Update user manual with the new functionalities
+
+App Store
+
+- [ ] [DIS] iTunes Connect: Create a new version following the `[major].[minor].[patch]`
+- [ ] [DIS] iTunes Connect: Update screenshots if needed
+- [ ] [DIS] Upload the binary to the App Store
+- [ ] [DIS] iTunes Connect: When release (manually, automatically, automatically not early than date)
+- [ ] [DIS] iTunes Connect: How to releae (immediately to all the users o 7-day period phased release)
+- [ ] [DIS] iTunes Connect: Reset iOS Summary Rating (Keep existing or reset the rating)
+- [ ] [DIS] iTunes Connect: Update changelogs
+- [ ] [DIS] iTunes Connect: Submit for review
+
+Git
+
+- [ ] [GIT] Merge branch `release/[major].[minor].[patch]` in master
+- [ ] [GIT] Create tag and sign it `[major].[minor].[patch]`
```

Comment:
Unless the commit should be signed with a digital signature, "sign" can be misunderstood here.

Unless that's what you head in mind, maybe change this to 

```
Tag release commit as "[major].[minor].[patch]"
```

Reply:
Yes that is the point. The commit of the creation of the tag should be signed to be used by our branding system